socs/agents/vantagepro2/drivers.py

```diff
@@ -312,6 +312,13 @@ def receive_data(self):
         loop_data['time_sunrise'] = byte_data[75]
         loop_data['time_sunset'] = byte_data[76]
 
+        # Correct UV Index by a factor of 10 and fix overflow
+        uvi = loop_data['UV']
+        if uvi < 0:
+            uvi += 2**7
```

Comment:
Great to see ... this should be 2**8 = 256, right?  i.e., -127 becomes +129.

Reply:
oops yes that makes sense